src/core/RenderTarget.js

```diff
@@ -149,6 +145,8 @@ class RenderTarget extends EventDispatcher {
 
 		}
 
+		this._setTextureOptions( options );
```

Comment:
I think we also must update `RenderTarget3D` since its texture reference is overwritten with a new instance of `Data3DTexture`.